src/basic/alloc-util.h

```diff
@@ -8,6 +8,10 @@
 
 #include "macro.h"
 
+#if HAS_FEATURE_MEMORY_SANITIZER
+#  include <sanitizer/msan_interface.h>
+#endif
+
```

Comment:
why after the "macro.h" inclusion? usually we place `<>` includes before `""` includes...

Reply:
Because HAS_FEATURE_MEMORY_SANITIZER is defined in macro.h.

Reply:
i wonder if we should move that macro definition into something meson adds to the gcc cmdline or such